zaza/openstack/charm_tests/test_utils.py

```diff
@@ -89,7 +89,53 @@ def audit_assertions(action,
             assert value == "PASS", "Unexpected failure: {}".format(key)
 
 
-class BaseCharmTest(unittest.TestCase):
+class CharmTestAssertions:
+    """Custom assertions to support zaza testings."""
+
```

Comment:
Not sure why it's a class?  Is it for consistency in terms of using `self.someassertion(...)` in a test file.

Reply:
On second thoughts these are useful outside of the test classes so I will move them.